tests/models/musicgen/test_modeling_musicgen.py

```diff
@@ -629,6 +629,12 @@ def get_mean_reldiff(failcase, x, ref, atol, rtol):
                                 if torch_device in ["cpu", "cuda"]:
                                     atol = atols[torch_device, enable_kernels, torch_dtype]
                                     rtol = rtols[torch_device, enable_kernels, torch_dtype]
+                                elif torch_device == "xpu":
+                                    # As of PyTorch 2.5 XPU backend supports only torch.nn.attention.SDPBackend.MATH
+                                    # which is implemented on PyTorch level using aten operators and is
+                                    # device agnostic with respect to implementation of each aten operator.
+                                    atol = atols["cuda", enable_kernels, torch_dtype]
```

Comment:
why `enable_kernels` not set to False here?

Reply:
I missed it. Fixed.